vademecum_VT.md

```diff
@@ -27,7 +27,10 @@ Firefox dispone di migliaia di componenti aggiuntivi, tra estensioni, temi, plug
 Esplora la realtà virtuale direttamente da Firefox, senza necessità di installare software specifici. Con WebVR la realtà aumentata e virtuale è gratuita, semplice e per tutti.
 
 #### Rust |QRcode|
-Un linguaggio di programmazione sviluppato per offrire un'esperienza unica, ma allo stesso tempo completa e complessa. E' un linguaggio di programmazione famigliare a chi già sa programmare, la sintassi è simile a quella del C.
+
+Rust è un linguaggio compilato con performace simili c ma garantento la safety del programma, sviluppato da Mozilla e componente fondamentale in firefox quantum.
```

Comment:
Rust è un linguaggio compilato con performance simili a quelle del C e garantisce, in aggiunta, la sicurezza del programma.
È sviluppato da Mozilla ed è un componente fondamentale in Firefox Quantum.